source/cart.html

```diff
@@ -20,11 +20,11 @@ <h1 class="page-title has-dash">{{ page.name }}</h1>
               <label for="item_{{item.id}}_qty">Quantity</label>
               {{ item | item_quantity_input, nil, 'option-quantity' }}
             </div>
-            <a class="cart-item-remove" title="Remove item" href="#">
+            <button title="Remove item: {{ item.product.name }}" class="cart-item-remove" aria-label="Remove item: {{ item.product.name }}" href="#">

```

Comment:
The `href="#"` should be removed on the button here, otherwise lgtm!

Reply:
yuup ty!